tjpcov_new/covariance_clusters.py

```diff
@@ -0,0 +1,611 @@
+from .covariance_builder import CovarianceBuilder
+import numpy as np
+import pyccl as ccl
+
+# Replace with CCL functions
+from scipy.integrate import quad, romb
+from scipy.special import gamma
+from scipy.interpolate import interp1d
+
+
+class CovarianceClusters(CovarianceBuilder):
+    """Covariance of Cluster + 3x2pt
+    version 1 , date: - ?? ?? 2019
+    Covariance matrix for a LSST-like case, using CCL packages
+    Evaluate here:
+    - NxN & NxCls (gg, gk, kk)
+    - Assuming full sky for now
+    - Included shot noise (1-halo term)
+    - Added full matrix in the end
+    TODO: verify the other limber auxiliary functions and new shot noise
+    """
+
+    # get c from CCL
+    c = 299792.458  # km/s
+    bias_fft = 1.4165
+    ko = 1e-4
+    kmax = 3  # TODO check if this is 3 or 4
+    N = 1024
+    overdensity_delta = 200
+
+    def __init__(self, config, ovdelta=200, survey_area=4 * np.pi):
```

Comment:
Add doc string

Reply:
Added it to the class rather than the constructor, since that's what will show in intellisense